### PR TITLE
eth/tracers: fix the issue of stateobject modified

### PR DESCRIPTION
**Pull request by jsvisa**
Signed-off-by: Delweng <delweng@gmail.com>

### CODE REVIEW

**Review by s1na — APPROVED** [review]
Ah good catch! The change looks good to me. I will wait for CI before merging.

File: eth/tracers/native/prestate.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func (t *prestateTracer) CaptureStart(env *vm.EVM, from common.Address, to commo
 
 	// The sender balance is after reducing: value and gasLimit.
 	// We need to re-add them to get the pre-tx balance.
-	fromBal := t.prestate[from].Balance
+	fromBal := new(big.Int).Add(common.Big0, t.prestate[from].Balance)
```
Comment (holiman) [inline eth/tracers/native/prestate.go:82]:
```suggestion
	fromBal := new(big.Int).Set(prestate[from].Balance))
```

**Review by holiman — APPROVED** [review]
LGTM

**Comment by holiman** [comment]
This was green, not sure why it wasn't repoorted here: https://ci.appveyor.com/project/ethereum/go-ethereum/builds/44908415

**Pull request merged**